.github/workflows/e2e-test.yml

```diff
@@ -41,9 +46,50 @@ jobs:
 
       - name: generate local SSH key
         run: ssh-keygen -t rsa -b 4096 -f ~/.ssh/id_rsa -N ""
+      
+      - name: Install Python deps
+        run: pip3 install requests wheel boto3
+
+      - name: Install go-junit-report
+        run: go install github.com/jstemmer/go-junit-report/v2@latest
 
       - name: run e2e test
-        run: make e2e
+        run: |
+          status=0
+          if ! make e2e > test_output.txt; then
+            echo "EXIT_STATUS=1" >> $GITHUB_ENV
+          fi
+            cat test_output.txt
         env:
           CLEANUP_TEST_LINODE_INSTANCE: true
           LINODE_TOKEN: ${{ secrets.DX_LINODE_TOKEN }}
+
+      - name: Copy report.xml from hack directory to root directory
```

Comment:
Can we skip this step? 
e.g.
`          filename=$(ls | grep -E 'report\.xml$')
          python tod_scripts/add_to_xml_test_report.py \
          --branch_name "${GITHUB_REF#refs/*/}" \
          --gha_run_id "$GITHUB_RUN_ID" \
          --gha_run_number "$GITHUB_RUN_NUMBER" \
          --xmlfile "hack/${filename}"`

Reply:
okay

Reply:
it won't work